buildbot

```diff
@@ -14,8 +14,8 @@ def echo_run(cmd):
 
 @contextmanager
 def instance():
-    arch = run(['uname', '-m'], stdout=PIPE).stdout.decode('latin1').strip()
-    vm = 'vm-{}'.format(arch).replace('_', '-')
+    # Unnecessary when there is only one VM, but allows yield and less changes
```

Comment:
I understand what you're doing here, and I think we can move this comment from the source code, into an extended git commit message. If anyone wants to dig into the history of that line, they can `git blame`, and should find that git commit message :)